.gitignore

```diff
@@ -5,8 +5,14 @@ vendor/
 screenshots/
 uploads/
 coverage/
+coverage-backend/
 dist/
 .DS_Store
 dump.rdb
 config.json
 npm-debug.log
+typings/
```

Comment:
The typings directory was in there because I had the node and express typings files for code completion in Visual Code